data-manager/jdbc/src/main/java/org/orbisgis/orbisdata/datamanager/jdbc/io/IOMethods.java

```diff
@@ -73,8 +72,8 @@ private static DriverFunction getDriverFromFile(@NotNull File file) {
             case "gpx":
                 return new GPXDriverFunction();
             default:
-                LOGGER.error("Unsupported file format.\n" +
-                        "Supported formats are : [shp, geojson, tsv, csv, dbf, kml, kmz, osm, gz, bz, gpx].");
+                LOGGER.error("Unsupported file format.\n"
+                        + "Supported formats are : [shp, geojson, tsv, csv, dbf, kml, kmz, osm, gz, bz, gpx].");
```

Comment:
Please keep the `+` a the end of the line.

Reply:
Automatic format put the + on the next line

Reply:
all the code has the + at the end of the line